README.md

```diff
@@ -23,87 +23,41 @@ npm install apollo-link-rest --save # or `yarn add apollo-link-rest`
 import { RestLink } from "apollo-link-rest";
 // Other necessary imports...
 
-// Create a RestLink for the Github API
-const link = new RestLink({ uri: "https://api.github.com" });
+// Create a RestLink for the REST API
+const restLink = new RestLink({
+  uri: 'https://swapi.co/api/',
+});
 
 // Configure the ApolloClient with the default cache and RestLink
 const client = new ApolloClient({
+  link: restLink,
   cache: new InMemoryCache(),
-  link
 });
 
-// A simple query to retrieve metada about a this repository
+// A simple query to retrieve data about the first person
 const query = gql`
-  query Repo {
-    repo @rest(type: "Repo", path: "/repos/apollographql/apollo-link-rest") {
-      id
+  query luke {
+    person @rest(type: "Person", path: "people/1/") {
       name
-      description
     }
   }
 `;
 
-// Invoke the query and log the response data
+// Invoke the query and log the person's name
 client.query({ query }).then(response => {
-  console.log(response.data.repo);
+  console.log(response.data.name);
 });
 ```
 
-[![Edit REST Link Basics](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/zkmnnxk5qp?expanddevtools=1&hidenavigation=1)
+[![Edit Basic Example](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/github/apollographql/apollo-link-rest/tree/master/examples/simple)
 
-### Apollo Client & React Apollo
```

Comment:
I'd revert this title change, and add a text description something like this

>Example of using Apollo Client with React […]

And maybe some code snippets from SearchShow.js? then say
> See more by viewing this in CodeSandbox: